src/main/java/com/vesoft/nebula/jdbc/impl/NebulaResultSetMetaData.java

```diff
@@ -32,30 +33,30 @@ public static NebulaResultSetMetaData getInstance(NebulaResultSet nebulaResultSe
     public static void release(){
         nebulaResultSetMetaData = null;
     }
+    
+    public List<String> getColumnNames() {
```

Comment:
if it's better to change the access to private?

Reply:
@TommyLemon